### DISCUSSION TITLE
Winston Customization

### DISCUSSION THREAD

**Discussion by 8BitAvenger** [Senryoku/Draftmancer#748] [Ideas]
A friend I play with regularly had an idea where Winston would have as many piles as the number of players +1. So if 3 are drafting, we'd have 4 piles. If 4, 5 piles, etc. I'm not sure this is good or bad, but would love to try. Would it be a reasonable ask to have a fill-in portion for # of piles, like we do for grid size in minesweeper for example?

**Comment by Senryoku** [comment]
Hello!

I'm not sure how practical it actually is, but this doesn't seem too bad to implement.
I deployed a prototype here: http://beta.draftmancer.com (PR: #749)
Please try it out and let me know if you find any issue! :)

**Reply by 8BitAvenger** [reply]
Tested it a touch myself just now with 5 stacks, 4 players (3 inprivate browsers) for a couple minutes and it seemed fine. I'll report back once I get a draft done w/my buddies. Thanks so much!
